.pre-commit-config.yaml

```
@@ -3,7 +3,9 @@ repos:
     rev: v4.0.1
     hooks:
     -   id: end-of-file-fixer
+        exclude: "test/fixtures/linter/.*.sql|plugins/sqlfluff-templater-dbt/test/fixtures/dbt/dbt_project/models/.*.sql|plugins/sqlfluff-templater-dbt/test/fixtures/dbt/templated_output/.*.sql|test/fixtures/templater/.*.sql"
     -   id: trailing-whitespace
+        exclude: "test/fixtures/linter/.*.sql|plugins/sqlfluff-templater-dbt/test/fixtures/dbt/dbt_project/models/.*.sql|plugins/sqlfluff-templater-dbt/test/fixtures/dbt/templated_output/.*.sql|test/fixtures/templater/.*.sql"
```

```suggestion
    -   id: end-of-file-fixer
        exclude: .*\.sql$
    -   id: trailing-whitespace
        exclude: .*\.sql$
```
Not sure why you didn't just use the regex I gave you